test/ci.jl

```diff
@@ -5,8 +5,8 @@ using TimeZones: TZData
 
 @testset "build process" begin
     # Clean out deps directories for a clean re-build
-    compiled_dir = TZData.compiled_dir()
-    tz_source_dir = TZData.tz_source_dir()
+    compiled_dir = TZData._compiled_dir(TZDATA_VERSION)
+    tz_source_dir = TZData._tz_source_dir(TZDATA_VERSION)
```

Comment:
I'll take another look at these tests tomorrow. I think there are some assumptions that are no longer true and possibly we can always run these tests instead of just in the CI environment

Reply:
Made a couple of small changes to the logic in these tests